src/less/widgets/widgets-general.less

```diff
@@ -199,8 +207,10 @@ a {
 .pf-subscription {
   form {
     overflow: hidden;
-    width: 100%;
-  
+    width: 80%;
```

Comment:
Ah. I can see why you did this, and then added 100% width to the slideout layout. I know we're not supporting folding just yet, but I think you'll find that that layout should also have 100% form width. It seems like it might make more sense to keep the general style 100%, and then just for the modal layout make it 80%? If that makes sense.